src/main/java/com/impactupgrade/nucleus/controller/MailchimpController.java

```diff
@@ -88,14 +89,50 @@ public Response messageEvent(
     return Response.status(200).build();
   }
 
+  // Audience events
+  @Path("/webhook/audience")
```

Comment:
Hey @VSydor, I'm not sure that we need a new endpoint. What I mean was the `messageEvent` endpoint (further up) is expecting JSON, but docs show that endpoint needing to accept form data as well. I just aimed a test webhook at a RequestBin and the "Campaign Sending" event came through as form data. But the docs are really confusing and show a mix of form data and JSON at the same time. Did you have any luck getting the message event to come through as JSON originally?

Reply:
![image](https://github.com/impactupgrade/nucleus-engine/assets/812741/c41d99df-691b-4c5b-950f-66769db3562c)

Just making sure we're talking about the same thing. Maybe the original Messages webhook came from a different/older setting?

Reply:
Argh, think I found it. https://mailchimp.com/developer/transactional/docs/webhooks/ Looks like way back in 2022, we initially set up the webhook to be based on Mailchimp **Transactional**, which isn't the standard marketing campaign emails. Transactional messages come from their Mandrill acquisition, which is positioned more like SendGrid. So we definitely need your new code, but need to expand it to get the Campaign event.

Reply:
>> Did you have any luck getting the message event to come through as JSON originally?
I think I was testing the code in a wrong way. I started with sending request to a request bin, then copied body payload to use it with Postman on my local env (assuming it was content-type/application-json). But you're right - the payload is sent as a form parameter indeed (json string as a form param "mandrill_events").

>> Just making sure we're talking about the same thing. Maybe the original Messages webhook came from a different/older setting?
We do talk about 2 diff webhooks (transactional from Mandrill vs marketing)

I was not able to get to testing audience webhook (not available for my free account). Could you please create a webhook from the account that you're using and share an example payload (json or form) with me so that I can use it to build the code and test on my local env?